packages/react-components/src/atoms/__tests__/MultiSelect.test.tsx

```diff
@@ -121,18 +121,15 @@ describe('invalidity', () => {
       />,
     );
     const input = getByRole('textbox');
-
     fireEvent.focusIn(input);
```

Comment:
The test description refers to the `right-click`, but the test uses `focusIn`. 
Does the bug affect other events (i.e. click)?

Reply:
I wasn't too sure on how to test it, so asserted on the regular behavior plus the right click. But I am open to suggestions